generators/typescript/utils/commons/src/codegen-utils/getSchemaOptions.ts

```diff
@@ -0,0 +1,11 @@
+import { SchemaOptions } from "@fern-typescript/zurg";
+
+export function getSchemaOptions({ allowExtraFields }: { allowExtraFields: boolean }): Required<SchemaOptions> {
```

Comment:
@dsinghvi I noticed that we actually set `skipValidation` to `false` and `breadcrumbsPrefix` to `[]` in _every_ instance of this function, so I included those properties here too (simply by specifying `Required<SchemaOptions>`. No `Pick` was necessary after all, but we can easily adapt this later to restrict specifics keys later if need be.

For what it's worth, users can still easier override the spread with any overrides they need like we normally do in migrations, i.e.

```typescript
return this.generatedRequestSchema
    .getReferenceToZurgSchema(context)
    .jsonOrThrow(referenceToParsedRequest, {
        ...getSchemaOptions({
            allowExtraFields: this.allowExtraFields
        }),
        skipValidation: true,
    });
```

Reply:
interesting do you know where we set `skipValidation` to true?

Reply:
i guess we only do it on response fields which this doesn't change